resources/language/resource.language.nl_nl/strings.po

```diff
@@ -693,6 +692,14 @@ msgctxt "#30930"
 msgid "This enables 'watch later' and 'continue watching' menus and syncs playback position between different devices using the same VRT NU account."
 msgstr "Dit activeert 'kijk later' en 'verderkijken' menu's and synchroniseert de afspeelpositie tussen verschillende toestellen die hetzelfde VRT NU account gebruiken."
 
+msgctxt "#30931"
+msgid "Enable Up Next (experimental)"
```

Comment:
I am not sure we want to add this optionally.
I mean if people installed Up Next, it is because they want to use this.
For me that is sufficient as "activation" (provided it works out-of-the-box).
So if we add this (e.g. to advertise this integration) I would enable it by default from the very start.

Reply:
I agree

Reply:
Dropped